apps/playground/project.json

```diff
@@ -44,13 +44,13 @@
           "budgets": [
             {
               "type": "initial",
-              "maximumWarning": "500kb",
-              "maximumError": "5mb"
```

Comment:
why are we increasing this threshold? is the change in the `switch.scss` not fixing the empty selector issue?

Reply:
That is just for the playground SPA. When built it was throwing a ton of warnings due to these. We don't actually publish this other than to the Github previews - so it isn't a big deal. I just upped them so that the warnings would go away.